reference/6/Microsoft.PowerShell.Utility/Send-MailMessage.md

```diff
@@ -0,0 +1,377 @@
+---
+ms.date:  06/09/2017
+schema:  2.0.0
+locale:  en-us
+keywords:  powershell,cmdlet
+online version:  http://go.microsoft.com/fwlink/?LinkId=821856
```

Comment:
Does this link have to be changed?

Reply:
No. There is a separate process outside of GitHub for managing those links.